tests/tests_service.yml

```diff
@@ -312,4 +312,4 @@
             name: linux-system-roles.firewall
           vars:
             firewall:
-              previous: replaced
+              - previous: replaced
```

Comment:
hmm - you may have found a bug in https://github.com/linux-system-roles/firewall/blob/master/tasks/main.yml#L65 - if the dict form is not working - I believe this may need to be 
```yaml
  loop: "{{ (firewall is mapping) | ternary([firewall], firewall) |
```
That is - the `|` takes precedence over the `is`

Reply:
This fails before it reaches the loop, I'll fix the cause but it is the when for that block of plays causing dict form not to fail, so its a bug I just created. I'll fix it.

Reply:
fixed. Left a dict form `previous: replaced` to make sure there is some coverage of dict form in the test suite now